pyms/flask/app/create_app.py

```diff
@@ -175,6 +188,12 @@ def init_metrics(self):
             )
             self.metrics.monitor(self.application)
 
+    def reload_conf(self):
+        self.delete_services()
+        self.config.reload()
+        self.init_services()
```

Comment:
@miguelgrubin an other aproach, we reload config but no restart services and app, @alexppg view that, if we restart the config we must restart the DDBB and other libs

Reply:
Yeah, it would be awesome to be able to add to this "restart" feature an abstract way to restart anything. This way, pyms wouldn't need to support every feature ever used but the developer could use it as a function, handler or something like that. @avara1986 @miguelgrubin